core/src/test/resources/HistoryServerExpectations/one_app_multi_attempt_json_expectation.json

```diff
@@ -11,6 +14,9 @@
     "completed" : true
   }, {
     "attemptId" : "1",
+    "startTimeEpoch" : 1426533910242,
+    "endTimeEpoch" : 1426533945177,
+    "lastUpdatedEpoch" : "",
```

Comment:
Oh I get it, it blanks the lastUpdated field so this is effectively not compared. (Would it just be easier to strip these?) Although I appreciate it doesn't matter in the sense that it's just for purposes of a diff, it seems like it's worth making the value it substitutes valid, like a 0. Or just removing it if possible.